app/controllers/admin/lost_utilities_controller.rb

```diff
@@ -0,0 +1,24 @@
+class Admin::LostUtilitiesController < Admin::BaseController
+  def create
+    @lost_utility = LostUtility.new lost_utility_params
+    if @lost_utility.save
+      utility = Utility.find @lost_utility.utility_id
+      utility_name = utility.name
+      utility_cost = utility.cost
+      flash[:success] = t "created"
+      render json:
+               {success: true, lost_utility: @lost_utility, utility_name:,
+                utility_cost:},
```

Comment:
```suggestion
               {success: true, lost_utility: @lost_utility, utility_name: utility.name,
                utility_cost: utility.cost},
```
sao ko viết thế này luôn đặt biến làm gì thế